src/foundry/client/apps/sidebar/apps/module-management.d.mts

```diff
@@ -45,19 +50,47 @@ declare global {
 
     override get isEditable(): boolean;
 
+    // TODO: Implement GetDataReturnType
     override getData(options?: Partial<Options>): MaybePromise<object>;
 
+    /**
+     * Given a module, determines if it meets minimum and maximum compatibility requirements of its dependencies.
+     * If not, it is marked as being unable to be activated.
+     * If the package does not meet verified requirements, it is marked with a warning instead.
+     * @param module  The module.
+     */
+    protected _evaluateDependencies(module: Module): void;
+
+    /**
+     * Given a module, determine if it meets the minimum and maximum system compatibility requirements.
+     * @param module  The module.
+     */
+    protected _evaluateSystemCompatibility(module: Module): void;
+
     override activateListeners(html: JQuery): void;
 
     protected override _renderInner(data: object): Promise<JQuery>;
```

Comment:
`data` works here; we should probably adjust the typing on this to be `ReturnType<this["getData"]>` across all formapps tbh